.github/copilot-instructions.md

```diff
@@ -191,6 +191,12 @@ See [docs/UITests.md](../docs/UITests.md) for detailed UI testing setup and usag
 4. **Network Management**: Offline mode and better error handling
 5. **Performance**: Image caching and smooth scrolling optimizations
 
+### Development Best Practices
+
+**Documentation and Research Guidelines:**
+- **Always search Microsoft Learn (MS Learn)** when implementing .NET MAUI features, gestures, animations, or any platform-specific functionality - Microsoft Learn provides the most up-to-date and official guidance for .NET MAUI development
+- Reference official documentation before implementing new features to ensure best practices and optimal performance
```

Comment:
There is an MCP server configured such as:
```json
{
  "mcpServers": {
    "microsoft-docs-mcp": {
      "type": "http",
      "url": "https://learn.microsoft.com/api/mcp",
      "tools": [ "*" ]
    }
  }
}
```
Can you update this to list the specific tool for usage? `microsoft_docs_search`?